.github/workflows/ci.yaml

```diff
@@ -936,15 +936,15 @@ jobs:
           . venv/bin/activate
           pip install mysqlclient sqlalchemy_utils
       - name: Run pytest (partially)
-        timeout-minutes: 10
+        timeout-minutes: 15
         shell: bash
         run: |
           . venv/bin/activate
           python --version
 
           python3 -X dev -m pytest \
             -qq \
-            --timeout=9 \
+            --timeout=20 \
```

Comment:
Should the pytest timeout not be smaller than the global timeout?

Reply:
What global timeout?

Reply:
I mean the job timeout.
Shouldn't we have `--timeout=20` less than `timeout-minutes: 15`

Reply:
`--timeout=20` means a single test case fails after 20 seconds
`timeout-minutes: 15` means the CI job fails after 15 minutes

Reply:
Those are not related.

The `--timeout=20` is seconds timeout for a single test, `timeout-minutes: 15` is in minutes for the entire CI job.

Reply:
seconds versus minutes! sorry for the noise.